config/purposes/tube_racks.yml

```diff
@@ -11,3 +8,9 @@ TR Stock 48:
 Heron Lysed Tube Rack:
   :asset_type: tube_rack
   :size: 96
+TR LRC Bank Seq:
```

Comment:
will these default to using the standard 96 plate printer label?

also now wondering whether the labware creator should be defined on the tubes or the tube racks purpose, maybe the racks should be set to labware creator `uncreatable`?

Reply:
Yes it will default to the standard 96 plate printer label. Is this something that needs changing? I haven't touched the existing implementation around that so assumed it would be the same.

In terms of the labware creator I think that will be covered in other stories which is why I have left it here. I think if a tube rack is being created it should be via the tube rack purpose but I'm not sure on the limitations with various handling various tube types within that tube rack. Happy to set it as un-creatable for the time being?